lib/src/util/useToggle.ts

```diff
@@ -27,5 +31,5 @@ export default function useToggle(initialValue: boolean): ReturnType {
     return newState;
   }
 
-  return [state, toggleState, setState];
+  return [state, toggleState, setState, setFalse, setTrue];
```

Comment:
Alternatively, now that this is getting more complicated maybe we should consider returning an object instead of an array?

Reply:
> Looks like most of the callers only call the setTrue option? Perhaps we should swap the setFalse and setTrue functions so you can just drop the , , from most calls?

They mostly call `setFalse`.  The `, ,` is to skip the `setState` function.  

> Alternatively, now that this is getting more complicated maybe we should consider returning an object instead of an array?

I was trying to avoid more breaking changes than necessary.  Though there are already breaking changes elsewhere in this PR, and `useToggle` is pretty new and probably minimally used outside of RSC, so it wouldn't be the end of the world (would make this PR a bit bigger though).  What do you think?

Reply:
> I was trying to avoid more breaking changes than necessary. Though there are already breaking changes elsewhere in this PR, and useToggle is pretty new and probably minimally used outside of RSC, so it wouldn't be the end of the world (would make this PR a bit bigger though). What do you think?

Actually, I've realized that `useDropdownEvents` is only calling the close handler when it's already checked that the dropdown is open.  So all the new `onClose` stuff probably isn't really necessary after all.  Removing it would make this whole PR simpler so I'll work on that.